[rospy.client][INFO] 2020-11-17 05:28:39,501: init_node, name[/train_10609_1605590919494], pid[10609]
[xmlrpc][INFO] 2020-11-17 05:28:39,502: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-17 05:28:39,502: Started XML-RPC server [http://luigi:41185/]
[rospy.init][INFO] 2020-11-17 05:28:39,503: ROS Slave URI: [http://luigi:41185/]
[rospy.impl.masterslave][INFO] 2020-11-17 05:28:39,503: _ready: http://luigi:41185/
[rospy.registration][INFO] 2020-11-17 05:28:39,504: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-17 05:28:39,505: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-17 05:28:39,603: registered with master
[rospy.rosout][INFO] 2020-11-17 05:28:39,604: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-17 05:28:39,610: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-17 05:28:39,611: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-17 05:28:39,634: topic[/odom] adding connection to [http://192.168.0.201:37487/], count 0
[rospy.internal][INFO] 2020-11-17 05:28:39,847: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-11-17 05:28:51,314: topic[/bev/flow_image] adding connection to [http://192.168.0.201:46457/], count 0
[rospy.internal][INFO] 2020-11-17 05:28:51,318: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:46457/], count 0
[rospy.core][INFO] 2020-11-17 05:28:55,531: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-11-17 05:28:55,538: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-17 05:28:55,539: topic[/bev/flow_image] removing connection to http://192.168.0.201:46457/
[rospy.internal][INFO] 2020-11-17 05:28:55,539: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:46457/
[rospy.internal][INFO] 2020-11-17 05:28:55,540: topic[/odom] removing connection to http://192.168.0.201:37487/
[rospy.impl.masterslave][INFO] 2020-11-17 05:28:55,540: atexit
[rospy.internal][WARNING] 2020-11-17 05:28:55,541: Unknown error initiating TCP/IP socket to 192.168.0.201:41905 (http://192.168.0.201:46457/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-11-17 05:28:55,542: Unknown error initiating TCP/IP socket to 192.168.0.201:47607 (http://192.168.0.201:37487/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

